pkg/model/constants.go

```diff
@@ -9,8 +9,9 @@ const (
 	PostgresqlDeploymentName           = ApplicationName + "-postgresql"
 	PostgresqlDeploymentComponent      = "database"
 	PostgresqlServiceName              = ApplicationName + "-postgresql"
-	PostgresqlImage                    = "postgres:12"
-	KeycloakImage                      = "jboss/keycloak"
+	PostgresqlImage                    = "postgres:9.5"
+	KeycloakImage                      = "jboss/keycloak:latest"
+	RHSSOImage                         = "registry.redhat.io/redhat-sso-7-tech-preview/sso-cd-openshift"
```

Comment:
I believe we should be pinning these down to a specific image. What we want is a specific version of the operator tied to a version of the operand. In this case keycloak or RHSSO.

Reply:
Makes sense, let's use tags here.